src/pages/Detail/Carousel.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const imgBox = [
+  {
+    id: 1,
+    path: './images/test_1.jpg',
+  },
+  {
+    id: 2,
+    path: './images/test_2.jpg',
+  },
+  {
+    id: 3,
+    path: './images/test_3.jpg',
+  },
+  {
+    id: 4,
+    path: './images/test_4.jpg',
+  },
+];
```

Comment:
- 상수는 상수라는 것을 명시하기 위해서 `UPPER_SNAKE_CASE` 로 작성해주세요!
```suggestion
const IMAGE_BOX = [
  {
    id: 1,
    path: './images/test_1.jpg',
  },
  {
    id: 2,
    path: './images/test_2.jpg',
  },
  {
    id: 3,
    path: './images/test_3.jpg',
  },
  {
    id: 4,
    path: './images/test_4.jpg',
  },
];
```
- 추가적으로 상수의 위치는 컴포넌트 코드 아래, 스타일 컴포넌트 위로 위치시켜 주시는 게 일반적 입니다!
```jsx
function Carousel (props) {
  ...
}

export default Carousel;

const IMAE_BOX = [ ... ];

const StyledComponentExample = styled.div`
  ...
`;
```

Reply:
확인! 수정완료했습니다 :)